error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -675,6 +675,34 @@ Flux<T> after(Mono<T> mono) {
     }
   }
 
+  /** Don't unnecessarily convert {@link Mono} to {@link Flux}. */
+  @SuppressWarnings("VoidMissingNullable")
+  static final class MonoFluxThen<T> {
```

Comment:
We haven't fully migrated yet, but the naming standard we're thinking of settling on would have this rule be named `MonoThen`, derived from the code in the `@AfterTemplate`.